TicketMS/Repositories/Implementation/EventRepository.cs

```diff
@@ -11,36 +12,35 @@ public EventRepository()
             _dbContext = new SpringDbContext();
         }
 
-        public int Add(Event @event)
+        public void Add(Event @event)
         {
-            throw new NotImplementedException();
+            _dbContext.Add(@event);
+            _dbContext.SaveChanges();
         }
 
-        public int Delete(int id)
+        public void Delete(Event @event)
         {
-            throw new NotImplementedException();
+                _dbContext.Remove(@event);
+                _dbContext.SaveChanges();
         }
 
-        public IEnumerable<Event> GetAll()
+        public async Task<IEnumerable<Event>> GetAll()
         {
-            var events = _dbContext.Events;
+            var events = await _dbContext.Events.ToListAsync();
             return events;
         }
 
-        public Event GetById(int id)
+        public async Task<Event> GetById(int id)
```

Comment:
It is a good pratice to add Async sufix to asynchronous methods. For example, here it should be named GetByIdAsync.

Reply:
This is applicable for all async methods.

Reply:
Done